src/clr-angular/button/button-group/_button-group.clarity.scss

```diff
@@ -10,23 +15,36 @@
     .btn {
       margin: 0;
       vertical-align: top;
+    }
 
+    .btn {
       &:not(:first-child) {
-        border-top-left-radius: 0;
-        border-bottom-left-radius: 0;
+        @include square-off-btngroup-btns(left);
       }
 
       &:not(:last-child) {
-        border-top-right-radius: 0;
-        border-bottom-right-radius: 0;
+        @include square-off-btngroup-btns(right);
       }
     }
 
-    &.btn-primary .btn:not(:last-child),
-    &.btn-success .btn:not(:last-child),
-    &.btn-danger .btn:not(:last-child),
-    &.btn-warning .btn:not(:last-child) {
-      margin: 0 $clr-default-borderwidth 0 0;
+    clr-tooltip {
```

Comment:
We don't style Angular components with element selectors, we target the class they add to the host. So here, you want `.tooltip`.

And I just realized we apparently don't have this as part of our coding guidelines, do you mind adding it with this PR? I would say it goes under "Typescript and Angular" > "Code style".

Reply:
Sure. No problem.